README.md

```diff
@@ -9,14 +9,13 @@ If no Dockerfile is found or provided in the provided application source, the fo
 
 - Uses the Oryx++ Builder to build the application source using [Oryx](https://github.com/microsoft/Oryx) to produce a
   runnable application image
-- Pushes this runnable application image to the provided Azure Container Registry
+- Pushes this runnable application image to the provided Container Registry
 - Creates or updates a Container App based on this image
 
 If a Dockerfile is found or discovered in the application source, the builder won't be used and the image will be built
 with a call to `docker build` and the Container App will be created or updated based on this image.
 
-If a previously built image has already been pushed to the ACR instance and is provided to this action, no application
-source is required and the image will be used when creating or updating the Container App.
+If a previously built image has already been pushed to a Container Registry and is provided to this action, no application source is required and the image will be used when creating or updating the Container App.

```

Comment:
nit: let's make sure lines in the `README.md` and `action.yaml` follow the 120 character limit where possible

Reply:
This comment still applies, but is a minor comment.